newspaper/configuration.py

```diff
@@ -34,6 +34,7 @@ def __init__(self):
         self.MAX_AUTHORS = 10  # num strings in list
         self.MAX_SUMMARY = 5000  # num of chars
         self.MAX_SUMMARY_SENT = 5  # num of sentences
+        self.EMPTY_PDF = "%PDF-"  # empty PDF constant
```

Comment:
ditto: see comment below, I don't think having a constant sentinel for empty pdfs is a good idea